### ISSUE TITLE
Multi-language support

### ISSUE THREAD

**Issue by marlun78** [marlun78/number-to-words#3]
Add pluggable languages files for multi-language support.

**Comment by thearabbit** [comment]
I would like to config `Khmer` language.
Please help me.

**Comment by marlun78** [comment]
Multi-lang support will be a part of the 2.x release, but it's not done yet.

**Comment by 102** [comment]
@marlun78 have you stopped to maintain the package?

**Comment by marlun78** [comment]
No, just been busy lately. I have a 2.0 in the making, but ETA. I'm guessing you were interested in some other language than English?

**Comment by 102** [comment]
@marlun78 yes, would be happy to help with Russian

**Comment by marlun78** [comment]
Sounds great! I'll see if I can share something this weekend.

**Comment by beeing** [comment]
Found a library that does similar to this with locale support.
https://github.com/Kibo/numbers2words

Perhaps can work on this to improve the languages.

**Comment by codesignist** [comment]
I am ready to help for Turkish.

**Comment by duduvien** [comment]
Hi, can I know is this still in consideration for releasing in next version?
